test/specs/crossHomePage.js

```diff
@@ -0,0 +1,30 @@
+const SELECTORS={
+  logoLink: '.logo',
+  howToLink: '.how-to-use',
+  infoLink: '.info',
+}
+
+class crossHomePage{
+  open(){
+    browser.url('http://localhost:3000/');
+  }
+
+ getElement(selectorName) {
+  return browser.$(SELECTORS[selectorName]);
+ }
+
+logo_page(){
```

Comment:
this function name is not very helpful. Maybe you mean something like `clickLogoButton`